### PR TITLE
feat(sidebar): Organization Stats chart

### PR DESCRIPTION
**Pull request by billyvg**
image

### CODE REVIEW

**Comment by billyvg** [comment]
@ckj is this something we still want?

**Comment by ckj** [comment]
@billyvg I think so, but I still think we'll still want to clean up the bar chart before we start featuring it so prominently. I wasn't able to make much progress on that front, but IIRC @Chrissy had a pretty compelling version rendered via svg.

@Chrissy is that still something that you want to introduce? If so, I think this could be a good reason to bump it up in terms of priority.

**Comment by Chrissy** [comment]
Yeah I agree. I'll move it up on the list.

File: src/sentry/static/sentry/app/components/sidebar/organizationStatsChart.jsx [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+}
+
+const OrganizationStatsChartWrapper = styled('div')`
+  font-size: 10px;
+  margin-top: 24px;
+  padding-left: 4px;
+  padding-right: 4px;
+`;
+
+const StyledBarChart = styled(BarChart)`
```
Comment (billyvg) [inline src/sentry/static/sentry/app/components/sidebar/organizationStatsChart.jsx]: Is this still needed?

**Comment by billyvg** [comment]
This should look better now due to https://github.com/getsentry/sentry/pull/8802

File: src/sentry/static/sentry/app/components/sidebar/organizationStatsChart.jsx [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+  renderLoading() {
+    return null;
+  }
+
+  renderBody() {
+    const {data} = this.state;
+    if (!data) return null;
+
+    let points = data.map(d => ({x: d[0], y: d[1]}));
```
Comment (Chrissy) [inline src/sentry/static/sentry/app/components/sidebar/organizationStatsChart.jsx]: i think we are supposed to use `series` moving forward?

File: src/sentry/static/sentry/app/components/sidebar/organizationStatsChart.jsx [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+    return (
+      <OrganizationStatsChartWrapper>
+        {t('Last 24h')}
+        <StyledBarChart points={points} label="events" height={60} />{' '}
```
Comment (Chrissy) [inline src/sentry/static/sentry/app/components/sidebar/organizationStatsChart.jsx]: i think we are supposed to use series moving forward?

File: src/sentry/static/sentry/app/components/sidebar/organizationStatsChart.jsx [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+      </OrganizationStatsChartWrapper>
+    );
+  }
+}
+
+const OrganizationStatsChartWrapper = styled('div')`
+  font-size: 10px;
+  margin-top: 24px;
```
Comment (Chrissy) [inline src/sentry/static/sentry/app/components/sidebar/organizationStatsChart.jsx]: lets get rid of some of these one-off values and use `space()` or `box` or something.

**Review by Chrissy — COMMENTED** [review]
couple small comments. the big thing rn is that scoped css. once we figure out what the intent was, then maybe we can find a better way to do it with props or composition or something.

File: src/sentry/static/sentry/app/components/sidebar/organizationStatsChart.jsx (hunk 1 above)
Comment (billyvg) [reply src/sentry/static/sentry/app/components/sidebar/organizationStatsChart.jsx]: I think we wanted more spacing between the bars

**Comment by billyvg** [comment]
@Chrissy not sure what's going on here:

image

**Comment by billyvg** [comment]
@ckj 	@Chrissy 	do we still want this?

image

**Comment by ckj** [comment]
It will be less necessary once we have cross-project issue graphs... I'm leaning towards scrapping it. Thoughts?

**Comment by billyvg** [comment]
Yeah I'd probably agree

**Pull request closed without merging**